src/components/userLogin_comp.js

```diff
@@ -0,0 +1,33 @@
+import { OnboardingHeader } from './onboardingHeader'
+
+
+function UserLoginComp() {
+return (
+    <form>
+        <OnboardingHeader h1={'Welcome back'} p ={'Enjoy extra features when you create an account with us.'}/>
+
+        <div className='flex justify-between items-center my-6'>
+        <button className='bg-gray-200 flex items-center py-2 px-10 mx-3 rounded-md'>Create Account with Google</button>
+        <button className='bg-gray-200 flex items-center py-2 px-10 mx-3 rounded-md'>Create Account with Github</button>
+        </div>
+        <div className='flex items-center'>
+        <div className='border-b-2 border-gray-200 w-full flex justify-center'></div>
+        <p className='flex justify-center px-3'>OR</p>
+        <div className='border-b-2 border-gray-200 w-full flex justify-center'></div>
+        </div>
+
+        <div className="mt-3">

```

Comment:
Remove the `mt-3` whereever they are. Let the `space-y-6` in the form control everything